src/main/java/seedu/progresschecker/ui/ProfilePanel.java

```diff
@@ -74,6 +74,7 @@ private String getTagColor(String tagName) {
         int index = getValueOfString(tagName) % TAG_COLORS.length;
         return TAG_COLORS[index];
     }
+    //@@author
```

Comment:
The following method `getValueOfString` after the `getTagColor` method was also written by me. Please help to end my collate info after `getValueOfString` method, thanks! 😃